setup_scripts/install_dependencies.sh

```diff
@@ -70,6 +70,12 @@ sudo apt-get install -y\
     clang-format\
     python-rosinstall
 
+echo "================================================================"
+echo "Installing Robotic Arm Dependencies
```

Comment:
```suggestion
echo "Installing Robotic Arm Dependencies"
```
Travis CI complained about this missing quotation mark.